SECURITY.md

```diff
@@ -0,0 +1,57 @@
+# Security Policy
+
+Quiet is an experimental, peer-to-peer, end-to-end encrypted, and serverless team chat application built on Tor & IPFS. While we aim to improve user autonomy and privacy beyond what centralized messengers can offer, **Quiet is not audited** and **should not be relied upon in high-risk scenarios requiring proven, thoroughly vetted security guarantees**.
```

Comment:
I would cut "and serverless" both because this has a separate meaning in industry and because we will be adding an optional server soon.